ext/drx/drx.c

```diff
@@ -1535,43 +1535,19 @@ get_scatter_gather_info(instr_t *instr, scatter_gather_info_t *sg_info)
 static reg_id_t
 simd_reg_to_zmm(reg_id_t simd_reg)
```

Comment:
No need for these functions anymore, please inline the reg_resize_to_ calls below and remove.

Reply:
Do you mean remove these functions? What about backward compatibility?

Reply:
They're static and not exposed.

Reply:
oh right... missed that

Reply:
Done